core/src/main/java/org/mskcc/cbio/portal/model/GeneticProfile.java

```diff
@@ -65,6 +66,7 @@ public GeneticProfile(String stableId, int cancerStudyId, GeneticAlterationType
       this.profileName = profileName;
       this.profileDescription = profileDescription;
       this.showProfileInAnalysisTab = showProfileInAnalysisTab;
+      this.referenceGenomeId = 1;
```

Comment:
it really does not matter, since later on importer will update this field based on the reference genome listed in the portal properties file. Can use a deprecated tag.

Reply:
added some comments to the new code. It will need some extra fixes.